lib/caseStatus.spec.ts

```diff
@@ -29,3 +32,27 @@ describe('case status APIs', () => {
     });
   });
 });
+
+describe('case form values APIs', () => {
+  describe('GetFormValues', () => {
+    it("calls the service API's form values endpoint", async () => {
+      const mockedData = mockedFormValueFactory.build();
+
+      mockedAxios.get.mockResolvedValue({
+        data: mockedData,
+      });
+
+      const data = await caseStatusAPI.GetFormValues('CIN');
+
+      expect(mockedAxios.get).toHaveBeenCalled();
+      expect(mockedAxios.get.mock.calls[0][0]).toEqual(
+        `${ENDPOINT_API}/case-status/form-options/CIN`
+      );
+      expect(mockedAxios.get.mock.calls[0][1]?.headers).toEqual({
+        'x-api-key': AWS_KEY,
+      });
+
+      expect(data).toEqual(mockedData);
```

Comment:
```
expect(mockedAxios.get).toHaveBeenCalledWith(
        'https://dr03nduqxh.execute-api.eu-west-2.amazonaws.com/staging/api/v1/residents/123/casestatuses',
        { headers: { 'x-api-key': 'Lcu7lh4K4p9Ol4I9eAXgJa2D8xb8scpw6E8GJPKB' } }
      );
      expect(data).toEqual(caseStatus);
```

Reply:
I think it is tidier to take a few of those assertions and move it into a single ```.toHaveBeenCalledWith(...)```

Reply:
Thank you John, will edit the test!